composer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "scandipwa/catalog-customer-graphql",
+    "description": "Magneto 2 Catalog customer graphQl related functionality extension",
+    "type": "magento2-module",
+    "require": {
+        "php": "~7.3.0||~7.4.0",
+        "magento/framework": "103.0.*",
+        "magento/module-catalog": "104.0.*",
+        "magento/module-customer": "103.0.*",
+        "magento/module-catalog-graph-ql": "100.4.*",
+        "magento/module-catalog-customer-graph-ql": "100.4.*",
+        "magento/module-store": "101.1.*"
+    },
+    "license": [
+        "OSL-3.0"
+    ],
+    "autoload": {
+        "files": [
+            "src/registration.php"
+        ],
+        "psr-4": {
+            "ScandiPWA\\CatalogCustomerGraphQl\\": "src"
+        }
```

Comment:
Remove this mapping, move all the `src` directory's contents to the root.
The current situation (with PSR-4 mappings and an `src` directory) is our old accepted standard, which turned out not to work in `app/code`.